lib/runtime-core/src/instance.rs

```diff
@@ -106,6 +108,22 @@ impl Instance {
             import_object: imports.clone(),
         };
 
+        // We need to push the code version so that the exception table can be read
+        // in the feault handler so that we can report traps correctly
+        #[cfg(unix)]
+        {
+            if let Some(msm) = instance.module.runnable_module.get_module_state_map() {
+                push_code_version(CodeVersion {
```

Comment:
We need to pop the code version when an `Instance` is `drop`ped. The code version won't be released otherwise, and there will be a memory leak.

Reply:
For sure -- thanks!  It seems the current system relies on `Instance`s being dropped in the opposite order that they were created, otherwise `pop_code_version` may pop the wrong one.  I think this is probably okay for now but it definitely seems like a bug